docs/cloud/cloud-databases/cloud-db-versions.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Cloud database versions
+layout: default
+parent: Manage databases
+grand_parent: Cloud
+nav_order: 4
+---
+
+# Additional database version information
+{: .no_toc }
+
```

Comment:
Add your include file here to act as an introduction to the content on the page:

```
{% include /cloud-db/cloud-summary-db-version.md %}
```